atom/browser/native_window_views_win.cc

```diff
@@ -90,6 +90,15 @@ bool NativeWindowViews::PreHandleMSG(
     // mode if it isn't already, always say we didn't handle the message
     // because we still want Chromium to handle returning the actual
     // accessibility object.
+    case WM_DWMCOLORIZATIONCOLORCHANGED: {
```

Comment:
@zcbenz I'm unfamiliar with that helper class so I'm probable doing something silly.  But while trying to do this

```cpp
CreateNamed(base::Bind(&SystemPreferences::HandleMessage, this), L"test");
```

I get this glorious error 

```bash
vendor\brightray\vendor\download\libchromiumcontent\src\base\memory\ref_counted.h(407): error C2039: 'Release': is not a member of 'atom::api::SystemPreferences'
```

Is there another helper that provides the required `Release` and `AddRef` methods that making a pointer to a non static member function requires?